ignition/model/lifecycle.py

```diff
@@ -21,23 +24,24 @@ def lifecycle_execute_response_dict(execute_response):
 
 class LifecycleExecution():
 
-    def __init__(self, request_id, status, failure_details=None, outputs=None, associated_topology=None):
+    def __init__(self, request_id, status, failure_details=None, outputs=None, associated_topology=None, version=None):
```

Comment:
Same as above, set LIFECYCLE_MESSAGE_VERSION as the default in the constructor?

Reply:
Addressed in commit 4.